commands/emoji.js

```diff
@@ -1,19 +1,60 @@
 const Augur = require("augurbot"),
-  u = require("../utils/utils");
+  u = require("../utils/utils"),
+  Jimp = require('jimp'),
+  emojiUnicode = require('emoji-unicode'),
+  svgToImg = require('svg-to-img'),
+  axios = require('axios');
 
 const Module = new Augur.Module()
 .addCommand({name: "emoji",
-  description: "Increases custom emoji size",
+  description: "Increases emoji size. Use multiple seperated by a space to combine them",
   syntax: ":emoji:",
-  info: "Displays full resolution emoji. Only enlarges emoji to full resolution uploaded to Discord.",
+  info: "Displays full resolution emoji. You can combine up to 25.",
   aliases: ["emote", "embiggen"],
   process: (msg, suffix) => {
     u.clean(msg, 0);
 
     let test = /<(a?):(\w+):(\d+)>/i;
-    let id = test.exec(suffix);
-    let ext = (id[1] ? ".gif" : ".png");
-    if (id) msg.channel.send({files: [{name: id[2] + ext, attachment: `https://cdn.discordapp.com/emojis/${id[3]}.${(id[1] ? "gif" : "png")}`}]});
+    let rows = [], cols = 1;
+    
+    for(x of suffix.split('\n')){
+      let j = [];
+      for(y of x.split(' ')) if(y != '' && y != ' ') j.push(y);
+      rows.push(j.join(' '));
+    }
```

Comment:
Scoping, changing var names for clarity
```suggestion
    for (let row of suffix.split('\n')) {
      let characters = [];
      for (let character of row.split(' '))
        if(character != '' && character != ' ') characters.push(character);
      rows.push(characters);
    }
```